.ci/benchmarks.groovy

```diff
@@ -46,6 +46,7 @@ pipeline {
         TF_VAR_public_key = "./id_rsa_terraform.pub"
         BENCHMARK_RESULT = "benchmark-result.txt"
         TFVARS_SOURCE = "system-profiles/8GBx1zone.tfvars" // Default to use an 8gb profile
+        USER = "benchci-${env.BUILD_ID}" // use build as prefix
```

Comment:
there are corner-cases if the same Build ID is used between executions, though IIUC, this won't be the case by default since it won't run on PRs but on branches. is that right?

Reply:
I hadn't considered that, but you're right, we cover 99% of the runs with this prefix. In practice, I don't think we're running the benchmarks in PRs right now.